client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx

```diff
@@ -225,13 +225,13 @@ class MappedDomainType extends React.Component {
 			);
 		} else {
 			primaryMessage = translate(
-				'Your domain mapping has not been setup. You need to update your nameservers, at the company you purchased your domain, to:',
+				'You domain mapping has not been set up. You need to update your name servers at the company where you purchased the domain to:',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.